### PR TITLE
chore: prevent bad formatting

### PR DESCRIPTION
**Pull request by alexandereneroth**
Runs a github action to detect commits with bad formatting.

Also formatted all files.

----

Base is react-wrappers implementation branch to prevent conflicts.

Should add a formatting commit when wrappers are done, so its possible to merge this.

### CODE REVIEW

**Comment by mJarsater** [comment]
Wouldn't this be better as a pre-commit hook?

**Comment by alexandereneroth** [comment]
> Wouldn't this be better as a pre-commit hook?

Wouldn't say it would be better, I feel like someone would circumvent the pre-commit hook, unknowingly or not.

I'd rather have both.

**Comment by mJarsater** [comment]
> > Wouldn't this be better as a pre-commit hook?
> 
> Wouldn't say it would be better, I feel like someone would circumvent the pre-commit hook, unknowingly or not.
> 
> I'd rather have both.

Alrightero! Just worried that this is gonna result in our PR containing files we weren't expecting when we created the PR. However I guess that's only gonna be the case before this have ran once, checkin all the files. 

Are you planning to add it to the tegel repo also?

**Comment by alexandereneroth** [comment]
> > > Wouldn't this be better as a pre-commit hook?
> > 
> > 
> > Wouldn't say it would be better, I feel like someone would circumvent the pre-commit hook, unknowingly or not.
> > I'd rather have both.
> 
> Alrightero! Just worried that this is gonna result in our PR containing files we weren't expecting when we created the PR. However I guess that's only gonna be the case before this have ran once, checkin all the files.
> 
> Are you planning to add it to the tegel repo also?

Although I'm not sure how you would circumvent the pre-commit hook. But I seem to remember Iv'e had people doing that in other repos. 

Maybe a pre-commit hook would be enough 🤔

I thought we could trial this here for a bit. But I'd love to have a pre-commit hook at least in the tegel repo.

**Comment by mJarsater** [comment]
> > > > Wouldn't this be better as a pre-commit hook?
> > > 
> > > 
> > > Wouldn't say it would be better, I feel like someone would circumvent the pre-commit hook, unknowingly or not.
> > > I'd rather have both.
> > 
> > 
> > Alrightero! Just worried that this is gonna result in our PR containing files we weren't expecting when we created the PR. However I guess that's only gonna be the case before this have ran once, checkin all the files.
> > Are you planning to add it to the tegel repo also?
> 
> Although I'm not sure how you would circumvent the pre-commit hook. But I seem to remember Iv'e had people doing that in other repos.
> 
> Maybe a pre-commit hook would be enough 🤔
> 
> I thought we could trial this here for a bit. But I'd love to have a pre-commit hook at least in the tegel repo.
> 
> (Added both here now)

Yeah, looks good here. But let's maybe start out with a pre-commit hook in the tegel repo?

File: .gitignore [hunk 1]
```diff
@@ -22,25 +22,3 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
```
Comment (mJarsater) [inline .gitignore]: Nice, this was my rouge changes from yesterday. Great that you removed them!

Comment (alexandereneroth) [reply .gitignore]: alright, where did they come from? 🤔 didnt understand how they ended up here after my rebase

Comment (mJarsater) [reply .gitignore]: It was when I was trying out amplify deployments using the amplify cli. It added some gitignore rules and they had to be merged to main inorder to deploy

**Pull request merged**